_maps/WIP/Sulaco_top.dmm

```diff
@@ -8213,17 +8221,14 @@
 /turf/open/floor,
 /area/sulaco/cargo/office)
 "qy" = (
-/obj/item/radio/intercom{
-	anyai = 1;
-	freerange = 1;
-	name = "General Listening Channel";
-	pixel_y = 30
+/obj/machinery/atmospherics/pipe/simple/yellow/hidden{
```

Comment:
Don't mapmerge with changes like this, simply just mass replace the map file like it was a .dm file